cargo-apk/README.md

```diff
@@ -73,6 +73,25 @@ version = 1
 name = "android.permission.WRITE_EXTERNAL_STORAGE"
 max_sdk_version = 18
 
+# See https://developer.android.com/guide/topics/manifest/queries-element#provider
+[[package.metadata.android.queries.provider]]
+authorities = "org.khronos.openxr.runtime_broker;org.khronos.openxr.system_runtime_broker"
+# Note: `name` attribute is not required for queries-provider however this is a workaround for aapt throwing errors about missing `android:name` attributes.
+name = "org.khronos.openxr"
+
+# See https://developer.android.com/guide/topics/manifest/queries-element#intent
+[[package.metadata.android.queries.intent]]
+actions = ["android.intent.action.SEND"]
+
+# See https://developer.android.com/guide/topics/manifest/queries-element#intent
+# Note: there can be several .data entries.
+[[package.metadata.android.queries.intent.data]]
+mime_type = "image/jpeg"
```

Comment:
For the intent bits, feel free to additionally link to the https://developer.android.com/guide/topics/manifest/intent-filter-element documentation, despite it already being mentioned at https://developer.android.com/guide/topics/manifest/queries-element#intent too.